build.zig

```diff
@@ -237,9 +258,9 @@ fn compileRaylib(b: *std.Build, target: std.Build.ResolvedTarget, optimize: std.
                 raylib.linkSystemLibrary("dl");
                 raylib.addIncludePath(.{ .cwd_relative = "/usr/include/libdrm" });
 
-                raylib.defineCMacro("PLATFORM_DRM", null);
-                raylib.defineCMacro("EGL_NO_X11", null);
-                raylib.defineCMacro("DEFAULT_BATCH_BUFFER_ELEMENT", "2048");
+                raylib.root_module.addCMacro("PLATFORM_DRM", "1");
+                raylib.root_module.addCMacro("EGL_NO_X11", "1");
+                raylib.root_module.addCMacro("DEFAULT_BATCH_BUFFER_ELEMENT", "2048");
```

Comment:
I wonder what this `DEFAULT_BATCH_BUFFER_ELEMENT` is. `src/config.h` mentions `RL_DEFAULT_BATCH_BUFFER_ELEMENTS` [here](https://github.com/raysan5/raylib/blob/5.0/src/config.h#L102) but it's a slightly different name, off by default, and not the same value.

@raysan5, do you have any ideas? Is this some old deprecated name for the setting? And does it really need to be defined when building for `-Dplatform=drm`? I don't see it mentioned by any other build script either.

Reply:
I think it's a deprecated name, it shouldn't be needed... I think... rlgl has a definition for it dependant on OpenGL version

Reply:
Yeah... I'll remove it. At a glance, it's the only suspicious config aside from the undocumented one at:
https://github.com/raysan5/raylib/blob/6610ab0dc73f87dc836526e0565e30d77faf0623/build.zig#L125
OpenGL deprecation... maybe that's some macOS thing.

Reply:
I think that just silences build warnings when calling OpenGL functions, which obviously is macOS specific, so it should be fine to only pass there.